browser/resources/settings/brave_routes.js

```diff
@@ -40,8 +40,9 @@ export default function addBraveRoutes(r) {
     r.BRAVE_WALLET = r.BASIC.createSection('/wallet', 'wallet')
     r.BRAVE_WALLET_NETWORKS = r.BRAVE_WALLET.createChild('/wallet/networks');
   }
-
-  r.BRAVE_HELP_TIPS = r.ADVANCED.createSection('/braveHelpTips', 'braveHelpTips')
+  if (!isGuest) {
+    r.BRAVE_HELP_TIPS = r.ADVANCED.createSection('/braveHelpTips', 'braveHelpTips')
+  }
```

Comment:
nit: It seems to work fine as is but I think we can do this similar with other cases.

```suggestion
  if (r.ADVANCED) {
    r.BRAVE_HELP_TIPS = r.ADVANCED.createSection('/braveHelpTips', 'braveHelpTips')
  } else if (!isGuest) {
     console.error('[Brave Settings Overrides] Could not find ACVANCED page', r)
  }
```

Reply:
5 min late 😹

Reply:
funny!  wrote same comment :)